src/widget.ts

```diff
@@ -438,3 +438,5 @@ export class ValueView extends DOMWidgetView {
     this.el.textContent = this.model.get('value');
   }
 }
+
+export class XTouchMiniView extends DOMWidgetView {}
```

Comment:
I am confused about how this worked in the past if `XTouchMiniView` did not exist but was set as the value of `_view_name`.

Reply:
I see, you don't instantiate that view in client code.